tests/ConfigTest.php

```diff
@@ -30,6 +30,7 @@ public function setUp(): void
             null,
             new Utilities
         );
+        parent::setUp();
```

Comment:
I'm curious about this. I'm noticing some of the test classes call parent in `setUp()`, and some don't. And the implementation in `PHPUnit_Framework_TestCase` is an empty function. Was this breaking before, or is it for style/correctness?

Reply:
this is mostly style/correctness piece. Since the base class is overriding the method, it should be calling its parent's setup as well. 

In this particular case, it should change anything functionally. (Since parent's setup is empty)